### ISSUE TITLE
Automagically download new images

### ISSUE THREAD

**Issue by crspybits** [crspybits/SharedImages#15]
When you re-launch the app, shouldn't it automagically refresh / pull latest images?
That would be my expectation as user.
and periodic auto-refresh to get new images when app is open.

**Comment by danligas** [comment]
Consider balancing this with energy consumption. My Samsung tells me when apps are active took often in background and encourages me to make them sleep to reserve power.

**Comment by crspybits** [comment]
Sounds like this is for apps in the background only @danligas. Is that right? I think the automagic download should occur only when the app is in the foreground. Sound reasonable?

**Comment by danligas** [comment]
Yes, limit update to foreground only. Badge should still advise with push
notification.

**Comment by crspybits** [comment]
Cool. Currently I'm not using push notifications. I'm experimenting. I find push notifications invasive. Instead, I'm using what iOS calls a "background fetch". As long as the app is in the background, it will get some run time periodically and check for newly available files. Shall see how this works.

**Comment by danligas** [comment]
Yes, I suspect that this would be easier to leverage in a serverless
scenario.

On Mon, Aug 7, 2017 at 9:07 PM, Christopher Prince <notifications@github.com
